### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by Luc4sguilherme**
Making a small improvement in the code and removing the assignment in conditional expressions.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (nolddor) [inline .eslintignore:1]: Is this file strictly necessary for the change to work? See [The .eslintignore File](https://eslint.org/docs/latest/use/configure/ignore#the-eslintignore-file)

> In addition to any patterns in the .eslintignore file, ESLint always follows a couple of implicit ignore rules even if the --no-ignore flag is passed. The implicit rules are as follows:
> - node_modules/ is ignored.
> - dot-files (except for .eslintrc.*) as well as dot-folders and their contents are ignored.

**Review by nolddor — COMMENTED** [review]
Hmm I have mixed feeling about this. Im aware that assign operators in conditional expressions practice is generally disallowed but there are valid reasons to use them.

I see this case as valid reason, as the code gets a bit more messy when not using it. So most likely I'm going to decline the PR. Anyway, thanks for the suggestion. 

Please check my comment about `.eslintignore` file usage. Still unsure if we need them and I missed something within the documentation.

File: .eslintignore (hunk 1 above)
Comment (Luc4sguilherme) [reply .eslintignore:1]: It really is no longer necessary to ignore the `node_modules` folder manually. The `eslint` seems to be doing this by default since 2014. https://github.com/btmills/eslint/commit/f0be1accf251f6acd2066bfc90705f4670e2146e

But most people don't know that. You can see this being done even in large projects. https://github.com/vercel/next.js/blob/canary/.eslintignore
https://github.com/webpack/webpack/blob/main/.eslintignore
https://github.com/mui/material-ui/blob/master/.eslintignore

Comment (nolddor) [reply .eslintignore:1]: We are fine then without having it at current code base.
Thanks for the clarification!
